cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -4251,6 +4251,51 @@
     </description>
   </property>
 
+  <!-- Retry configuration for task worker -->
+  <!-- Use an exponential delay that cap at 2 seconds interval -->
+  <property>
+    <name>system.worker.retry.policy.base.delay.ms</name>
+    <value>10</value>
```

Comment:
nit: 10ms is too short IMO. I'd bump it to 100ms.

Reply:
ack, done